### ISSUE TITLE
Translate Learning Path pages navigation

### ISSUE THREAD

**Issue by tsadler1988** [InnerSourceCommons/InnerSourceLearningPath#333]
Translate 'Back to Learning Path homepage', 'Next', and 'Previous' on non-English Learning Path pages.

**Comment by rrrutledge** [comment]
Yeah, good point.

**Comment by lenucksi** [comment]
@tsadler1988 says the following on this: "If you get me the translated words I can get them onto Jekyll and rendered!"

@lenucksi says: 
-  "Zurück zur Homepages des Learning Path"
- "Nächstes Kapitel"
- "Vorheriges Kapitel"
are the German words in the order as listed in the opening ticket.

**Comment by rrrutledge** [comment]
This will be interesting to see!

**Comment by tsadler1988** [comment]
- Added 'Thank you to this article's contributors!' as this is also hardcoded in English.
- @tsadler1988 to look at the implementation.
- @lenucksi to reach out for other translations.

**Comment by lenucksi** [comment]
Highlighting our Chinese, Japanese and Italian contributors here.
It would be much appreciated if you could, just as a ticket comment here, provide translations for the following strings so we can properly built them into our rendered site improving the translation there.

##Strings

-  'Back to Learning Path homepage',  
- 'Next',
-  'Previous',
-  'Thank you to this article's contributors!' 

# Example: 
## German
-  'Back to Learning Path homepage',  -> "Zurück zur Homepages des Learning Path"
- 'Next', -> "Nächstes Kapitel"
-  'Previous', -> "Vorheriges Kapitel"
-  'Thank you to this article's contributors!' -> "Wir danken für die Mitarbeit an diesem Artikel:"

Chinese: @yvonneT1, @adadai88, @niminghong, @jupeng198262, @WillemJiang, @adadai88, @tanzhongyi003
Japanese: @ystk @johtani @ShinobuSaito
Italian:  @OpenPj


Of course we're happy about any other language too - even if we might not yet have the articles translated. 😄

**Comment by OpenPj** [comment]
### Italian
* 'Back to Learning Path homepage', -> "Torna alla home del Learning Path"
* 'Next', -> "Prossimo"
* 'Previous', -> "Precedente"
* 'Thank you to this article's contributors!' -> "Grazie ai contributori di questo articolo!"

**Comment by ystk** [comment]
### Japanese

- 'Back to Learning Path homepage', -> 'Learning Pathのホームに戻る'
- 'Next', -> '次へ'
- 'Previous', -> '前へ'
- 'Thank you to this article's contributors!' -> 'この文書に貢献いただいた方々'

**Comment by WillemJiang** [comment]
### Chinese
* 'Back to Learning Path homepage', -> '回到Learning Path主页'
* 'Next', -> '下一页'
* 'Previous', -> '上一页'
* 'Thank you to this article's contributors!' -> '感谢你对这篇文章的贡献'

**Comment by jupeng1982629** [comment]
> Translate 'Back to Learning Path homepage', 'Next', 'Previous', 'Thank you to this article's contributors!' on non-English Learning Path pages.

'Thank you to this article's contributors!' this sentence is a little confusing to me.
Is it mean "Thank you for your contribution on this article"?

At least, according to chinese version from @WillemJiang above on my understanding means this.
But Japanese version from @ystk 义武，on my understanding means "对此文章做过贡献的人们" in chinese，Although, I know Japanese usually express connotation, so that also can be understand as appresiation.

Germany version is at same meaning as chinese version.

So, May I suggest to use  "Thank you for your contribution on this article" as English version?

**Comment by jupeng1982629** [comment]
Italian version means "Thanks to the contributors of this article!" translated by google.
if it is translated in chinese, we will say "感谢此文的贡献者"
Italian version is more solemn. : ) 
@OpenPj

**Comment by ystk** [comment]
Thanks for your feedback, @jupeng1982629 .
Japanese translation includes the meaning of "Thanks" in the word of "いただいた".  As mentioned, Japanese usually express connotation.

**Comment by jupeng1982629** [comment]
@ystk 义武san，My japanese is not good when leaning and working on it, nearly ten years ago, Now I return them to my japanese teacher. : P
ni hon go wa sin se ci da. : )

**Comment by lenucksi** [comment]
That's a really interesting discussion. Also, impressive language skills @jupeng1982629 👍 

If we go with @jupeng1982629's suggestion this becomes "Wir danken Ihnen für die Mitarbeit an diesem Artikel" if translated literally to German. 
To me this sounds, at least in my German translation, a bit like it would address the reader of the article not honor the contributors and translators of the specific article.
"Wir danken für die Mitarbeit an diesem Artikel:" would literally translate to "We thank for the contribution to this article:" for English.

Also, thanks a lot for all the translations! 🎉 @OpenPj @ystk @WillemJiang @jupeng1982629

**Comment by jupeng1982629** [comment]
>That's a really interesting discussion. Also, impressive language skills @jupeng1982629 👍

@lenucksi Thank you, : ), Berlin is a big capital, but what a pity, I haven't be there for have a look. But I worked in Munich and Hamburg for several mouths for BMW&Harman, 6 years ago, It's so impressive, I like German city and people there so much.

> "Wir danken für die Mitarbeit an diesem Artikel:"

@lenucksi Hi，Johonnes，I just noticed that there is a ":", not "!", at the end of sentence. Does it mean there would be some names of contributers following after this ":"? if so, in my opinion, "感谢此文的贡献者" would be more suitable for chinese version.

@WillemJiang Hi, Willem, How do you think?

**Comment by lenucksi** [comment]
> > "Wir danken für die Mitarbeit an diesem Artikel:"
> 
> @lenucksi Hi，Johonnes，I just noticed that there is a ":", not "!", at the end of sentence. Does it mean there would be some names of contributers following after this ":"? if so, in my opinion, "感谢此文的贡献者" would be more suitable for chinese version.

Yes, the `:` is intentional and was meant exactly as you suggested. I'll add that into the original comment of me, thanks for the highlight & question.

**Comment by jupeng1982629** [comment]
@lenucksi You're welcome.

**Comment by tsadler1988** [comment]
Thank you, danke, grazie, ありがとうございました, and 谢谢 everyone! I've raised https://github.com/InnerSourceCommons/innersourcecommons.org/pull/237 with the translations you've provided (I've used @WillemJiang 's original comment for Chinese, but it can easily be changed).
